test/harvest/ansible/harvest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
```

Comment:
harvest should not fail, but there is a flaky for shared memory that sometimes has a small bytes difference. Adding retries for now.

Reply:
would make sense to use instead assert.Eventuallyf in the tests with problems? this way we avoid rerunning them all and also will pass locally while using go test command

Reply:
I don't think it would help in this case. For some reason the growth of the shared mem is not exactly the same in every run :(

Reply:
Just checked and the test is already using Eventually